runtime/src/accounts_db.rs

```diff
@@ -4103,7 +4103,7 @@ impl AccountsDb {
 
     /// Scan through all the account storage in parallel
     fn scan_account_storage_no_bank<F, B>(
-        snapshot_storages: &[SnapshotStorage],
+        snapshot_storages: &[&Vec<Arc<AccountStorageEntry>>],
```

Comment:
For all these similar changes, is there a reason to not use `&[&SnapshotStorage]` instead?

Reply:
arguing with clippy that wants [] instead of vec